src/main/java/Duke.java

```diff
@@ -1,10 +1,153 @@
+                printList(commandStorage);
+            }else if (line.contains("done")) {
+                updateTaskStatus(commandStorage, line);
+            }else if (line.contains("todo")) {
+                addToDo(line);
+            }else if (line.contains("deadline")) {
+                addDeadline(line);
+            }else if (line.contains("event")) {
+                addEvent(line);
+            }
+        } while (line.equalsIgnoreCase("bye") == false);
+    }
+
+    public static void addTaskToList(Task t) {
+        commandStorage[Task.numOfTasks] = t;
+        Task.numOfTasks++;
+
+        printSeparation();
+        System.out.println("    " + "Got it. I've added this task: ");
+
+        System.out.format("    ");
+        printTaskDescription(t);
+
+        System.out.format("    Now you have %d task%s in the list.%n", Task.numOfTasks,
+                (Task.numOfTasks == 1) ? "" : "s");
+        printSeparation();
+    }
+
+    public static void addToDo(String line) {
+        int endOfTypeIndex = 4;
+        String taskName = line.substring(endOfTypeIndex+1);
+        ToDo t = new ToDo(taskName);
+        addTaskToList(t);
+    }
+
+    public static void addDeadline(String line) {
+        int endOfTypeIndex = 8;
+
+        int endOfTaskNameIndex = line.indexOf('/', endOfTypeIndex) - 1;
+        String taskName = line.substring(endOfTypeIndex+1, endOfTaskNameIndex);
+
+        int startOfTaskTimeIndex = line.indexOf("/by") + 4;
+        String taskTime = line.substring(startOfTaskTimeIndex);
+
+        Deadline d = new Deadline(taskName, taskTime);
+
+        addTaskToList(d);
+    }
+
+    public static void addEvent(String line) {
+        int endOfTypeIndex = 5;
+
+        int endOfTaskNameIndex = line.indexOf('/', endOfTypeIndex) - 1;
+        String taskName = line.substring(endOfTypeIndex+1, endOfTaskNameIndex);
+
+        int startOfTaskTimeIndex = line.indexOf("/at") + 4;
+        String taskTime = line.substring(startOfTaskTimeIndex);
+
+        Event e = new Event(taskName, taskTime);
+
+        addTaskToList(e);
+    }
+
+    private static void printTaskDescription(Task t) {
+        switch (t.getTaskType()) {
+        case "T":
+            System.out.println("[" + t.getTaskType() + "][" + t.getStatusIcon() + "] " + t.getTaskName());
+            break;
+        case "D":
+            System.out.println("[" + t.getTaskType() + "][" + t.getStatusIcon() + "] " + t.getTaskName() + " (by: "
+                    + t.getTaskTime() + ")");
+            break;
+        case "E":
+            System.out.println("[" + t.getTaskType() + "][" + t.getStatusIcon() + "] " + t.getTaskName() + " (at: "
+                    + t.getTaskTime() + ")");
+            break;
+        }
+    }
+
+    public static void printList(Task[] commandStorage) {
+        printSeparation();
+        if (Task.numOfTasks==0) {
+            System.out.println("    Dude, the list is empty! o_O");
+        }else {
```

Comment:
Maybe you could add a space after the closing curly brackets } and then type "else"